containers/unit-tests/build

```diff
@@ -7,7 +8,10 @@ if test -z "${docker:=$(which podman || which docker || true)}"; then
   exit 1
 fi
 
-$docker build --build-arg debian_arch=amd64 --build-arg personality=linux64 -t cockpit/unit-tests ${dir}
-$docker tag  cockpit/unit-tests:latest docker.io/cockpit/unit-tests:latest
-$docker build --build-arg debian_arch=i386 --build-arg personality=linux32 -t cockpit/unit-tests:i386 ${dir}
-$docker tag  cockpit/unit-tests:i386 docker.io/cockpit/unit-tests:i386
+if [ -z "${1:-}" ] || [ "${1:-}" = amd64 ]; then
```

Comment:
You could simplify that to `[ "${1:-amd64}" = amd64 ]`.  Arguably less readable, though.